component-api/src/main/java/org/talend/sdk/component/api/service/discovery/DiscoverDataset.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2006-2020 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.api.service.discovery;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.talend.sdk.component.api.meta.Documentation;
+import org.talend.sdk.component.api.service.ActionType;
+
+/**
+ * Mark a method as retruning the {@link DiscoverDatasetResult} of a datastore. The only configuration
+ * parameter will be an {@link org.talend.sdk.component.api.configuration.Option} named "datastore"
```

Comment:
the name is dependent to service's method and its parameter's name or `@Option("name")` (if applied) so, you should remove this in javadoc.

Reply:
ok